superflore/utils.py

```diff
@@ -148,7 +148,7 @@ def trim_string(string, length=80):
 
 
 def get_license(l):
-    bsd_re = '^(BSD)((.)*([124]))?'
+    bsd_re = '^(BSD)((.)*([124]))?|([1234])(.)?Clause(.)*(BSD)'
```

Comment:
@allenh1 My regex skills are pretty dusty, I tried to make it as safe as possible... so I just added a second option to match, the original one is untouched.